WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostPublishSettingsViewModel.kt

```diff
@@ -162,13 +165,24 @@ class EditPostPublishSettingsViewModel @Inject constructor(
             return
         }
         siteModel?.let {
-            _showJetpackSocialContainer.value = true
-            val state = if (showNoConnections()) {
-                jetpackUiStateMapper.mapNoConnections(::onJetpackSocialConnectProfilesClick)
+            if (showNoConnections()) {
+                val shouldShowJetpackSocialNoConnections =
+                    appPrefsWrapper.getShouldShowJetpackSocialNoConnections(it.id, JetpackSocialFlow.POST_SETTINGS)
```

Comment:
IINM the `getShouldShowJetpackSocialNoConnections` seems to be using the `SHOULD_HIDE_JETPACK_INSTALL_FULL_PLUGIN_CARD` key, that to me it seems related to another feature. Is that intended? 🤔 ...also, the function naming is about showing the no connections, while the key is about hiding something (that might be confusing I guess). Wdyt?

Reply:
Good catch, thanks! I had copy + pasted this method and forgot to replace the key 😅